docs/.wordlist.txt

```diff
@@ -47,3 +47,9 @@ UI
 UUID
 VM
 YAML
+DSS
+JupyterLab
+MLFlow
+GPUs
+runtime
+MicroK
```

Comment:
You should put those in `.custom_wordlist.txt`. That way, you can update `.wordlist.txt` from the starter pack without any conflicts and keep your custom terms separate.

But that's just a convention. It will still work this way. :)